sos/python/Sos.pxd

```diff
@@ -878,7 +878,7 @@ cdef extern from "dsos.h":
     dsos_query_t dsos_query_create(dsos_container_t cont)
     void dsos_query_destroy(dsos_query_t query)
     int dsos_query_select(dsos_query_t query, const char *clause)
-    char *dsos_query_errmsg(dsos_query_t query)
+    const char *dsos_query_errmsg(dsos_query_t query)
```

Comment:
@nichamon, did you check this with the older cython/build tools?

Reply:
Yes, I tested with Cython version 0.29.21, where there is no problem with and without this patch.